nav/dataset/dataset_pds3_cassini_iss.py

```
@@ -97,37 +138,64 @@ def _extract_img_number(img_name: str) -> int:
+    @staticmethod
+    def _results_path_stub(volume: str, filespec: str) -> Path:
+        f"""Get the results path stub for an image filespec.
+
+        Parameters:
+            volume: The volume name.
+            filespec: The filespec of the image.
 
         Returns:
-            The mapped filename.
+            The results path stub. This is {volume}/{filespec} without the .IMG extension.
+        """
```

Ruff (0.14.0)

f-string used as docstring. Python will interpret this as a joined string, rather than a docstring.

(B021)